roofit/roofitcore/inc/RooAbsCollection.h

```diff
@@ -439,6 +440,8 @@ class RooAbsCollection : public TObject, public RooPrintable {
 #error "Please remove this unneeded code."
 #endif
 
+  bool replaceImpl(const RooAbsArg& var1, const RooAbsArg& var2);
```

Comment:
The function is called `replace` but it takes two const references as arguments, that was odd to read. But I see this is already a public API so we can't change it at this point

Reply:
Thanks a lot for the review!

This method is a member function of the `RooAbsCollection`, which is a glorified `std::vector<RooAbsArg*>`.

The args that are replaced are `const` because they are not changed, it's only the collection that is changed.